examples/school/views/StudentRemove.js

```diff
@@ -0,0 +1,66 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { graphql, compose } from 'react-apollo';
+
+import Formsy from 'formsy-react';
+import Select from '../components/Select';
+import { removeStudent } from '../graphql/students';
```

Comment:
maybe change remove for only remove, the context is in the import '../graphql/students'